src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -209,32 +210,30 @@ private async Task<ListResponseModel<OrganizationUserUserDetailsResponseModel>>
         var readAllAuthorized = (await _authorizationService.AuthorizeAsync(
             User, new OrganizationScope(orgId), OrganizationUserUserDetailsOperations.ReadAll)).Succeeded;
 
-        if (!readAllAuthorized)
+        if (readAllAuthorized)
         {
-            var readWithAccountRecovery = (await _authorizationService.AuthorizeAsync(
-                User, new OrganizationScope(orgId), OrganizationUserUserDetailsOperations.ReadAccountRecovery)).Succeeded;
+            var results = await _organizationUserUserDetailsQuery.Get(request);
+            var responseList = results
+            .Select(result => new OrganizationUserUserDetailsResponseModel(result.OrgUser, result.TwoFactorEnabled, result.ClaimedByOrganization))
+            .ToList();
 
-            if (!readWithAccountRecovery)
-            {
-                throw new NotFoundException();
-            }
+            return (new ListResponseModel<OrganizationUserUserDetailsResponseModel>(responseList));
```

Comment:
Unresolved earlier comment: https://github.com/bitwarden/server/pull/5756#discussion_r2069790516
(unless dotnet format is doing this in which case it's fine)